### PR TITLE
WIP Put impact summary in separated xml file.

### PR DESCRIPTION
**Pull request by ismailsunni**
WIP Put impact summary in separated xml file.

### CODE REVIEW

**Comment by timlinux** [comment]
LGTM thanks @ismailsunni just two questions:

1. Can you check that when doing a ``Layer`` -> ``Save as`` operation that the summary table is correctly copied.
2. Can you clarify if in the metadata xml a relative path is saved to the summary table? I know we can do this implicitly, but I think it is better to do it explicitly so that e.g. if you are using a third party metadata viewer you can see a link and click through to the summary table.

**Comment by ismailsunni** [comment]
> Can you check that when doing a Layer -> Save as operation that the summary table is correctly copied.

yes, it is copied.

> Can you clarify if in the metadata xml a relative path is saved to the summary table? I know we can do this implicitly, but I think it is better to do it explicitly so that e.g. if you are using a third party metadata viewer you can see a link and click through to the summary table.

So, we store the path to the html file in the xml, right?. Ok.

Btw, this PR is on hold, since in our last discussion we want to do the generation of the report in impact function.  Anyway, I need to wait for @Gustry 's refactor first.

File: Makefile [hunk 1]
```diff
@@ -185,7 +185,7 @@ testdata:
 	@echo "Updating inasafe_data - public test and demo data repository"
 	@echo "Update the hash to check out a specific data version        "
 	@echo "------------------------------------------------------------"
```
Comment (timlinux) [inline Makefile]: This is a great change

**Comment by timlinux** [comment]
Hi @ismailsunni yes agreed - holding out for the html generator will be much cleaner, thanks!

**Comment by ismailsunni** [comment]
I think this PR is no longer needed since we have generated html report from json.

**Pull request closed without merging**